core-processor/src/executor.rs

```diff
@@ -132,18 +147,13 @@ pub(crate) fn charge_gas_for_instantiation(
     gas_allowance_counter: &mut GasAllowanceCounter,
 ) -> Result<(), ExecutionErrorReason> {
     let amount = gas_per_byte * code_length as u64;
-
     log::trace!("Charge {} for module instantiation", amount);
```

Comment:
such trace may be also included in `charge_gas` fn using debug formatter for gas operation

Reply:
Super idea! Done.